src/components/Deployment/Deployment.tsx

```diff
@@ -0,0 +1,232 @@
+  right: 28px;
+  border: none;
+  cursor: pointer;
+  border-radius: 10px;
+  height: 30px;
+  width: 80px;
+`;
+
+interface Account {
+  status: string;
+  value?: any;
+  title?: string;
+  address?: string;
+  createdAt?: Date;
+  state?: string;
+  store?: string;
+  type?: string;
+  updatedAt?: Date;
+}
+
+const Deployment = () => {
+  const [deployedChains, setDeployedChains] = useState<Account[]>([]);
+  const [undeployedChains, setUndeployedChains] = useState<Account[]>([]);
+  const [confirmModal, setConfirmModal] = useState<boolean>(false);
+  const [selectedChain, setSelectedChain] = useState<number | null>(null);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+
+  const { getSdkForChainId, accountAddress } = useEtherspot();
+  const theme: Theme = useTheme();
+
+  const fetchDeploymentData = async (chainId: number) => {
+    const sdk = getSdkForChainId(chainId);
+    if (accountAddress && sdk) {
+      try {
+        await sdk.computeContractAccount();
+        let account = await sdk.getAccount();
+        return account;
+      } catch (err) {
+        //
+      }
+    }
+  };
+
+  const getDeployementStatus = async () => {
+    try {
+      const result = await Promise.all(
+        supportedChains.map(async ({ chainId, title, iconUrl }) => {
+          const account = await fetchDeploymentData(chainId);
+          if (!account) return;
+          return { ...account, chainId, title, iconUrl };
+        })
+      );
+      const accounts = result.filter((acc) => acc);
+      return Promise.allSettled(accounts).catch(() => {
+        return [];
+      });
+    } catch (err) {
+      return [];
+    }
+  };
+
+  const hideModal = () => {
+    setConfirmModal(false);
+  };
+
+  const deploymentData = async () => {
+    const data: Account[] | undefined = await getDeployementStatus();
+    if (!data) return;
+    const [unDeployed, deployed] = data.reduce(
+      (acc: Account[][], curr: Account) => {
+        if (curr?.value?.state === 'UnDeployed') {
+          acc[0].push(curr);
+        } else {
+          acc[1].push(curr);
+        }
+        return acc;
+      },
+      [[], []]
+    );
+    setUndeployedChains(unDeployed);
+    setDeployedChains(deployed);
+  };
+
+  const deploy = async () => {
+    if (!selectedChain || !accountAddress) return;
+    const sdk = getSdkForChainId(selectedChain);
+    if (!sdk) return;
+    try {
+      await deployAccount(sdk);
+    } catch (err) {
+      setErrorMessage('Failed to proceed with selected actions!');
+      hideModal();
+    }
+  };
+
+  const handleClose = () => {
+    setErrorMessage('');
+  };
+
+  useEffect(() => {
+    deploymentData();
+  }, []);
+
+  return (
+    <Card title="Deployments" marginBottom={20}>
+      {errorMessage && <ErrorMessage errorMessage={errorMessage} onClose={handleClose} />}
+      {!!confirmModal && (
+        <Modal zIndex>
+          <Paragraph>Deploy your wallet on Etherium to sign messages</Paragraph>
```

Comment:
1. Ethereum is the network
2. Ether is the currency

always remember ✌️ 

```suggestion
          <Paragraph>Deploy your wallet on Ethereum to sign messages</Paragraph>
```